src/Microsoft.Health.TaskManagement/JobHosting.cs

```diff
@@ -200,6 +201,7 @@ public static async Task<string> ExecuteJobWithHeartbeatsAsync(IQueueClient queu
 
             var jobInfo = new JobInfo { QueueType = queueType, Id = jobId, Version = version }; // not other data points
 
+            // WARNING: Avoid using 'async' lambda when delegate type returns 'void'
```

Comment:
Is there a better way we should be making these timers?

Reply:
Probably it would be to use something like: https://learn.microsoft.com/en-us/dotnet/api/system.threading.periodictimer?view=net-8.0